tmt/utils.py

```diff
@@ -1275,6 +1275,20 @@ def yaml_to_dict(data: Any,
     return loaded_data
 
 
+def yaml_to_list(data: Any,
+                 yaml_type: Optional[YamlTypType] = None) -> List[Any]:
+    """ Convert yaml into list """
+    yaml = YAML(typ=yaml_type)
```

Comment:
I believe that safer for tmt would be do `yaml_type or "safe"`

Reply:
I edited `yaml_to_*` functions and put there implicit parameter `'safe'`